src/main/kotlin/day11/DefaultFileOperations.kt

```
@@ -0,0 +1,22 @@
+    override fun copy(sourcePath: String, targetPath: String) {
+
+        try {
+            val sourceFile = File(sourcePath)
+            val targetFile = File(targetPath)
+            val copyText = sourceFile.readText()
+
+            targetFile.writeText(copyText)
+        } catch (e: FileNotFoundException) {
+            println("파일을 찾을 수 없습니다")
+        } catch (e: Exception) {
+            println(Exception())
```

detekt (1.23.7)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)